operations/mimir-tests/test-helm-parity-generated.yaml

```diff
@@ -506,6 +506,9 @@ spec:
         - -ingester.max-global-metadata-per-metric=10
         - -ingester.max-global-metadata-per-user=30000
         - -ingester.max-global-series-per-user=150000
+        - -memberlist.bind-port=7946
+        - -memberlist.join=dns+gossip-ring.default.svc.cluster.local:7946
+        - -overrides-exporter.ring.store=memberlist
```

Comment:
This is not correct. Shouldn't be set if the overrides-exporter ring is disabled.